packages/openneuro-app/src/scripts/types/event-types.ts

```diff
@@ -61,60 +74,73 @@ export const mapRawEventToMappedNotification = (
   const {
     type,
     resolutionStatus,
-    status: eventStatus,
     requestId,
-    targetUserId,
+    targetUserId: eventTargetUserId,
+    target,
     reason,
   } = event
 
-  let title = note || "General Notification"
-  let mappedType: MappedNotification["type"] = "general"
+  let title = "General Notification"
+  const mappedType: MappedNotification["type"] = type
   let approval: MappedNotification["approval"]
   let requesterUser: User | undefined
   let adminUser: User | undefined
-  const capitalize = (str: string) =>
-    str ? str.charAt(0).toUpperCase() + str.slice(1) : str
+  let targetUser: User | undefined
+  let needsReview = false
+  let resStatus = resolutionStatus
 
   switch (type) {
     case "contributorRequest":
-      title = `[${type}] User Contributor Request for Dataset`
-      mappedType = "approval"
+      title = "is requesting to be added to"
+      requesterUser = user // user initiating the request (admin)
+      targetUser = target || rawNotification.user // fallback
+      approval = resolutionStatus ?? "pending"
+      needsReview = approval === "pending"
+      break
+    case "contributorCitation":
+      title = "is requesting"
+      adminUser = user
+      // fallback to rawNotification.user if target is missing (but not the admin)
+      targetUser = target ||
+        (rawNotification.user && rawNotification.user.id !== user.id
+          ? rawNotification.user
+          : {
+            id: eventTargetUserId || "",
+            name: "Unknown",
+            email: "",
+            orcid: "",
+          })
       approval = resolutionStatus ?? "pending"
-      requesterUser = user
+      needsReview = approval === "pending"
       break
-    case "contributorResponse":
-      title = `[${type}] User Contributor ${
-        capitalize(eventStatus ?? "")
-      } for Dataset`
-      mappedType = "response"
-      approval = eventStatus as "accepted" | "denied"
+
+    case "contributorRequestResponse":
+      title = "responded to a contributor request"
       adminUser = user
+      targetUser = target
+      approval = resolutionStatus ?? "pending"
       break
-    case "contributorCitation": {
-      mappedType = "citationRequest"
-      const status: "pending" | "accepted" | "denied" =
-        (resolutionStatus as "pending" | "accepted" | "denied") ?? "pending"
-      approval = status
-      const targetName = event.target?.name || "Unknown User"
-      if (status === "pending") {
-        title =
-          `[${type}] An admin has requested ${targetName} be added as an Author`
-      } else {
-        const capitalizedStatus = status.charAt(0).toUpperCase() +
-          status.slice(1)
-        title = `[${type}] ${targetName} has ${capitalizedStatus} authorship`
-      }
+
+    case "contributorCitationResponse":
+      title = "had their citation responded to"
```

Comment:
```suggestion
      title = "had their citation request for"
```
I think this wording would help.